### PR TITLE
feat(content): Navy Questioning / Human–Hai related missions adjusted

### PR DESCRIPTION
**Pull request by bene-dictator**
**Content/Fix (Missions)**

## Summary
Adjusted the missions:
- "Human to Hai Space"
- "Hai Wormhole Warning"
- "Pact Questioning"

"Human to Hai Space – No Contact" and "Hai Wormhole Warning" no longer offer if the Hai-Human Treaty is announced. 

"Human to Hai Space" offers if you have either met the Hai, or if the Hai-Human treaty has been announced. The guy will still sound a little clueless but I like it.

"Hai Wormhole Warning" and "Pact Questioning" are both landing missions. Their dialogue has been adjusted to suit this better. I'm looking for feedback on this particularly.

## Save File
This save file can be used to play through the new mission content:
- Go to the Hai and come back to human space for "Hai Wormhole Warning".
- Help the Pact in Pact Recon for "Pact Questioning", then go to a Near Earth / Paradise planet.
Both should be farely easy to test. 

## To-do
- [x] Tested Hai Wormhole Warning
- [x] Tested Pact Questioning
- [ ] Tested Pact Questioning while doing Katya's missions
- [ ] Tested "Human to Hai Space"

### CODE REVIEW

**Comment by Anarchist2** [comment]
Assigning Derpy, since he wrote Wormhole warning and FW rework.

File: data/human/human missions.txt [hunk 1]
```diff
@@ -4409,10 +4414,11 @@ mission "Hai Wormhole Warning"
 			has "First Contact: Unfettered: declined"
 		not "Block Hai Wormhole Warning: offered"
 		not "Wanderers: Alpha Surveillance E: offered"
+		not "event: hai-human resolution announced"
 	on offer
 		log "Was pulled aside by a Navy officer and told to keep the wormhole to the Hai a secret. The Republic doesn't want the knowledge of the wormhole to be widespread, as that could cause chaos for both the Republic and the Hai."
 		conversation
-			`Upon entering the spaceport, you're stopped by a Navy officer. "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on a landing pad behind you.`
+			`When you open the door to let in the Navy officer on inspection duty, you expect him to produce the usual set of landing papers and bureaucracy. But instead he says, "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on the landing pad next to the <ship>.`
```
Comment (Amazinite) [inline data/human/human missions.txt:4421]:
```suggestion
			`When you open the door to let in the Navy officer on inspection duty, you expect him to produce the usual set of landing papers and bureaucracy. But instead he says, "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on the landing pad next to <ship>.`
```

File: data/human/human missions.txt [hunk 2]
```diff
@@ -4477,7 +4483,7 @@ mission "Hai Wormhole Warning"
 			label ok
 			`	"That's good to hear," the officer responds. "Thank you for your cooperation, <last>. Now, if you'll please depart my ship so that I can be on my way."`
 			label end
-			`	The officer leads you out of the ship and closes the doors behind you. As you make your way back to the spaceport, you turn back and see the Frigate take off, slowly getting smaller in the sky until it finally disappears.`
+			`	The officer leads you out of the ship and closes the doors behind you, while the actual Navy officer on inspection duty stands beside the <ship>, brandishing a set of landing documents. When you've finished signing them, you turn back to see the Frigate take off, slowly getting smaller in the sky until it finally disappears.`
```
Comment (Amazinite) [inline data/human/human missions.txt:4486]:
```suggestion
			`	The officer leads you out of the ship and closes the doors behind you, while the actual Navy officer on inspection duty stands beside <ship>, brandishing a set of landing documents. When you've finished signing them, you turn back to see the Frigate take off, slowly getting smaller in the sky until it finally disappears.`
```

File: data/human/human missions.txt [hunk 3]
```diff
@@ -4409,10 +4414,11 @@ mission "Hai Wormhole Warning"
 			has "First Contact: Unfettered: declined"
 		not "Block Hai Wormhole Warning: offered"
 		not "Wanderers: Alpha Surveillance E: offered"
+		not "event: hai-human resolution announced"
```
Comment (Amazinite) [inline data/human/human missions.txt:4417]:
```suggestion
		not "Hai Leaks Response 1: offered"
```

**Pull request merged**